scout/server/blueprints/panels/views.py

```diff
@@ -73,6 +98,8 @@ def panels():
         panel_names=panel_names,
         panel_versions=panel_versions,
         institutes=institutes,
+        search_string=search_string,
```

Comment:
This string doesn't show in the search form after performing the search. Perhaps avoid passing it to the template?

Reply:
Ahh, I was looking into the form field, to be precompiled with the term from the previous search!